lib/resources/r4/document_reference.yaml

```diff
@@ -16,9 +262,45 @@ fields:
       system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/72
       info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#code-set-72-clinical-event-codes
 
+- name: type
+  required: 'No'
+  type: CodeableConcept
+  action:
+  - create
+  description: Physician Admission Note
```

Comment:
Why the change from 'Precise type of clinical document' to 'Physician Admission Note'? here and below.

Reply:
I just changed it because it was the docRef that I did locally and to add the LOINC coding

Reply:
Fixed